src/wallet/test/wallet_tests.cpp

```diff
@@ -50,7 +50,9 @@ BOOST_FIXTURE_TEST_CASE(rescan, TestChain100Setup)
         AddKey(wallet, coinbaseKey);
         WalletRescanReserver reserver(&wallet);
         reserver.reserve();
-        BOOST_CHECK_EQUAL(nullBlock, wallet.ScanForWalletTransactions(oldTip, nullptr, reserver));
+        const CBlockIndex* stop_block = nullptr;
+        BOOST_CHECK_EQUAL(wallet.ScanForWalletTransactions(oldTip, nullptr, reserver, stop_block), CWallet::ScanResult::SUCCESS);
+        BOOST_CHECK_EQUAL(nullBlock, stop_block);
```

Comment:
pIndexStop was nullptr, so this should be tip instead?

Reply:
@MarcoFalke `stop_block` is only set if the scan failed, it isn't set to tip if the scan succeeds